### PR TITLE
Get informations such name and email from http Headers

### PR DESCRIPTION
**Pull request by gallak**
finally pull request only concern name 
mail was insert by PR#4585
hope it's ok

### CODE REVIEW

File: config.default.php [hunk 1]
```diff
@@ -198,9 +198,19 @@
...
+define('REVERSE_PROXY_GET_FIELDS', FALSE);
+
 // Header name to use for the username
 define('REVERSE_PROXY_USER_HEADER', 'REMOTE_USER');
 
+// Header name to use for the common name  ( need REVERSE_PROXY_GET_FIELDS to True)
+define('REVERSE_PROXY_NAME_HEADER', 'REMOTE_CN');
+
+//Header name to use for the mail ( need REVERSE_PROXY_GET_FIELDS to True)
+define('REVERSE_PROXY_MAIL_HEADER', 'REMOTE_MAIL');
```
Comment (fguillot) [inline config.default.php:211]: This new option seems to be a duplicate of `REVERSE_PROXY_EMAIL_HEADER` defined below at line 208.

File: app/User/ReverseProxyUserProvider.php [hunk 2]
```diff
@@ -133,7 +135,11 @@ public function getUsername()
      */
     public function getName()
     {
-        return '';
+        if (! REVERSE_PROXY_NAME_HEADER ){
+            return '';
+	}else{
+            return $this->name;
+	}
```
Comment (fguillot) [inline app/User/ReverseProxyUserProvider.php:142]: Do you mind indenting your code properly? It must be consistent with the rest of the code.

**Review by fguillot — CHANGES_REQUESTED** [review]

File: config.default.php (hunk 1 above)
Comment (gallak) [reply config.default.php]: scorie of myself ( I re used REVERSE_PROXY_EMAIL_HEADER in code)

File: app/User/ReverseProxyUserProvider.php (hunk 2 above)
Comment (gallak) [reply app/User/ReverseProxyUserProvider.php]: tab replace by space (need to find tips about code indent in terminal)

Comment (fguillot) [reply app/User/ReverseProxyUserProvider.php]: the `else` have to be surrounded by 1 space (for all your changes). Look at the GitHub diff.

**Comment by fguillot** [comment]
Closing stale pull-request

**Pull request closed without merging**